lib/src/excel.dart

```diff
@@ -12,13 +12,36 @@ import 'util.dart';
 /// Service class that provides dart functions to interact with Excel.
 @Injectable()
 class ExcelDart {
-  static const _startAddress = 'A1';
-  static final _startAddressInt = 'A'.codeUnitAt(0);
+  static const _tableColumnStartAddress = 'A';
+  static const _tableRowStartAddress = 4;
+  static const _thresholdAddress = '\$B2';
```

Comment:
can you include a screenshot? I think itll help illustrate the pr

Reply:
Yep, sorry, that would make much more sense! See [here](https://github.com/googleinterns/dv360-excel-plugin/pull/53#issue-448467208)